static/js/ReaderPanel.jsx

```diff
@@ -1404,6 +1404,11 @@ class ReaderDisplayOptionsMenu extends Component {
     const hasEnglish = !!data.text.length;
     return !(hasHebrew && hasEnglish);
   }
+  renderPunctuationToggle() {
```

Comment:
I think this is more clear as `shouldPunctuationToggleRender`, so it doesn't give the idea that function is performing the render.